firmware/ventilator-controller-stm32/Core/Inc/Pufferfish/Driver/Serial/Backend/Frames.h

```diff
@@ -17,14 +17,17 @@ namespace Pufferfish::Driver::Serial::Backend {
 
 struct FrameProps {
   static const size_t payload_max_size = 254;
-  static const size_t chunk_max_size = payload_max_size + 2;  // including delimiter
+  static const size_t chunk_max_size = payload_max_size + 2;    // including delimiter
+  static const size_t encoded_max_size = payload_max_size + 1;  // including cobs
   using ChunkBuffer = Util::ByteVector<chunk_max_size>;
+  using EncodedBuffer = Util::ByteVector<encoded_max_size>;
```

Comment:
maybe we should be reusing `ChunkBuffer ` here instead of using a different size for `EncodedBuffer`, I maybe wrong but I think we are missing the case where include_delimiter is true, https://github.com/pez-globo/pufferfish-software/pull/281#discussion_r588073273